src/tests/JIT/opt/Compares/conditionalIncrements.cs

```diff
@@ -25,6 +25,15 @@ static byte cinc_byte_min_max(byte op1)
         return op1 >= 43 ? byte.MinValue : byte.MaxValue;
     }
 
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    static byte cinv_byte_min_max(byte op1)
```

Comment:
```suggestion
    static byte cinc_byte_min_max(byte op1)
```

Reply:
These are the `cinv` tests that emit `cinc`. 🙂